### ISSUE TITLE
like button is cut off

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#878]
```
blame:
http://code.google.com/p/html5rocks/source/browse/www.html5rocks.com/static/css/
base.css#734

.page header .share li.googleplus { left: 0; }
.page header .share li.facebook   { left: 85px; width: 47px; overflow: hidden; }
.page header .share li.twitter    { left: 158px; }

make that simpler:

.page header .share li.googleplus { left: 0; }
.page header .share li.facebook   { left: 85px; }
.page header .share li.twitter    { left: 180px; }

```

Original issue reported on code.google.com by `sst...@gmail.com` on 1 May 2012 at 11:42

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision a99ff408f8d0.
```

Original comment by `ericbide...@html5rocks.com` on 9 May 2012 at 7:09
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks!
```

Original comment by `ericbide...@html5rocks.com` on 9 May 2012 at 7:09
